src/BodyFunctionality/Body.java

```diff
@@ -0,0 +1,298 @@
+package BodyFunctionality;
+
+import javafx.scene.paint.Color;
+
+import java.util.ArrayList;
+
+
+public class Body
```

Comment:
Should probably implement bodypartinterface

Reply:
I plan on doing that next.